HPS/Heartland/view/adminhtml/templates/form/cc.phtml

```diff
@@ -1,151 +1,285 @@
-                    class="admin__control-select admin__control-select-month validate-cc-exp required-entry">
-                <?php foreach ($block->getCcMonths() as $k => $v): ?>
-                    <option value="<?php echo $block->escapeHtml($k); ?>"
-                            <?php if ($k == $ccExpMonth): ?>selected="selected"<?php endif ?>>
-                        <?php echo $block->escapeHtml($v); ?>
-                    </option>
-                <?php endforeach; ?>
-            </select>
-            <select id="<?php /* @noEscape */ echo $code ?>_expiration_yr" name="payment[cc_exp_year]"
-                    class="admin__control-select admin__control-select-year required-entry">
-                <?php foreach ($block->getCcYears() as $k => $v): ?>
-                    <option value="<?php /* @noEscape */ echo $k ? $block->escapeHtml($k) : '' ?>"
-                            <?php if ($k == $ccExpYear): ?>selected="selected"<?php endif ?>>
-                        <?php echo $block->escapeHtml($v); ?>
-                    </option>
-                <?php endforeach ?>
-            </select>
+            <div class="fields group group-2">
+                <div class="field no-label month">
+                    <div class="control">
+                        <input type="hidden" id="<?php /* @noEscape */echo $code ?>_expiration"
+                               name="payment[cc_exp_month]" />
+                    </div>
+                </div>
+                <div class="field no-label year">
+                    <div class="control">
+                        <input type="hidden" id="<?php /* @noEscape */echo $code ?>_expiration_yr"
+                               name="payment[cc_exp_year]" />
+                    </div>
+                </div>
+            </div>
         </div>
     </div>
 
-    <?php if ($block->hasVerification()): ?>
         <div class="field-number required admin__field _required">
-            <label class="admin__field-label" for="<?php /* @noEscape */ echo $code; ?>_cc_cid">
-                <span><?php echo $block->escapeHtml(__('Card Verification Number')); ?></span>
-            </label>
             <div class="admin__field-control">
-                <input type="text" title="<?php echo $block->escapeHtml(__('Card Verification Number')); ?>"
-                       class="required-entry validate-cc-cvn admin__control-cvn admin__control-text"
-                       id="<?php /* @noEscape */ echo $code; ?>_cc_cid"
-                       name="payment[cc_cid]" value="<?php /* @noEscape */ echo $block->getInfoData('cc_cid') ?>"/>
+                <input type="hidden" title="<?php echo $block->escapeHtml(__('Card Verification Number')); ?>"
+                       id="<?php /* @noEscape */
+                       echo $code; ?>_cc_cid"
+                       name="payment[cc_cid]" value="<?php /* @noEscape */
+                echo $block->getInfoData('cc_cid') ?>"/>
             </div>
         </div>
```

Comment:
Above, why are we storing card numbers and CVNs? We won't have access to this information and should never be stored/passed to the merchant server.